data/mapcss-mapping.csv

```diff
@@ -458,7 +458,7 @@ highway|living_street|tunnel;[highway=living_street][tunnel?];;name;int_name;457
 deprecated|deprecated;458;x
 deprecated|deprecated;459;x
 deprecated|deprecated;460;x
-sport|handball;461;
+deprecated|deprecated;461;x
```

Comment:
Wow, we **can't** change numeric values and move existing tags! Return handball back here :)